docs/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Version 3.40.0-eisop3 (November ??, 2023)
 
 **User-visible changes:**
 
+The nullness checker now issues redundant test against null warnings for switch statements when the expression is
```

Comment:
This is not just for switch statements, also for switch expressions. Please make sure there is or to add test coverage for both.
Try to simplify the sentence.

Reply:
> This is not just for switch statements, also for switch expressions. Please make sure there is or to add test coverage for both. Try to simplify the sentence.

Thanks, I have updated it and please see whether you like it.